.editorconfig

```diff
@@ -49,21 +50,311 @@ indent_size = 2
 # Sort using and Import directives with System.* appearing first
 dotnet_sort_system_directives_first = true
 # Avoid "this." and "Me." if not necessary
-#dotnet_style_qualification_for_field = false:suggestion
-#dotnet_style_qualification_for_property = false:suggestion
-#dotnet_style_qualification_for_method = false:suggestion
-#dotnet_style_qualification_for_event = false:suggestion
+dotnet_style_qualification_for_field = false:suggestion
+dotnet_style_qualification_for_property = false:suggestion
+dotnet_style_qualification_for_method = false:suggestion
+dotnet_style_qualification_for_event = false:suggestion
 
 csharp_style_inlined_variable_declaration = false:suggestion
```

Comment:
Some properties are duplicated, like this one, some cleanup is required.

In this case, it is set to `false` here and to `true` below.

Reply:
That's because I copied the settings from .NET and then exported the Resharper settings, which also generated some for dotnet style. 

I can't see the one that is true. Both occurrences are false for csharp_style_inlined_variable_declaration

Reply:
I cleaned them up!